src/project.cc

```diff
@@ -138,12 +151,11 @@ Project::Entry GetCompilationEntryFromCompileCommandEntry(
 
     // Cleanup path for previous argument.
     if (next_flag_is_path) {
-      arg = cleanup_maybe_relative_path(arg);
-
+      std::string arg1 = cleanup_maybe_relative_path(arg);
       if (add_next_flag_to_quote_dirs)
-        config->quote_dirs.insert(arg);
+        config->quote_dirs.insert(arg1);
```

Comment:
Instead of `arg1` (which doesn't convey any extra meaning) use something a bit more descriptive like `normalized_path` or similar.